unitTests/ui/SimpleFilterTest.ts

```diff
@@ -224,5 +228,90 @@ export function SimpleFilterTest() {
         expect($$(aSimpleFilter.cmp.root).findAll('.coveo-dropdown-background').length).toEqual(1);
       });
     });
+
+    describe('with clear button functionality', () => {
+      let simpleFilterWithEraser: Mock.IBasicComponentSetup<SimpleFilter>;
+      let simpleFilterContainer: Dom;
+
+      function clickOnClearButton() {
+        $$(simpleFilterContainer.find('.coveo-simplefilter-eraser')).trigger('click');
+      }
+
+      function selectSomeValues() {
+        simpleFilterWithEraser.cmp.selectValue('foo');
+        simpleFilterWithEraser.cmp.selectValue('bar');
+      }
+
+      function getClearButton() {
```

Comment:
If we hoist this function up one level and adjust the implementation to use `cmp.element`, we could reuse it in the test on line 199?